docs: add livepeerjs reference (#215)
livepeer/livepeer-studio-docs: docs/02-quickstart/index.mdx

With this, you're ready to start building!

### 2. Start Building {#start-building}

:::info

> Looking to use a SDK? Check out [Livepeer.js](https://livepeerjs.org/)!

Reviewer: wdyt?
```suggestion
Looking for an SDK? Check out [Livepeer.js](https://livepeerjs.org/)!
```

> :::

Reviewer: I don't actually know this `:::` syntax, but just wanna point out that the 2 new ones are inconsistent: the other one has `note` on the closing tag, while this one only has the `:::`. Which should be fixed?

Author: The `:::` syntax let us use [admonitions](https://docusaurus.io/docs/markdown-features/admonitions) so we can add notes or caution messages.

I was thinking of using note for prerequisite assumptions and info for more information related to the current content
